idea/src/META-INF/plugin.xml

```diff
@@ -2985,6 +2985,15 @@ The Kotlin plugin provides language support in IntelliJ IDEA and Android Studio.
                      language="kotlin"
     />
 
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.inspections.SuspiciousCollectionReassignmentInspection"
+                     displayName="Suspicious collection reassignment by augmented assignment"
```

Comment:
I don't like this description. It says nothing to normal user. I think we should say something about list under the hood, at least.